app/components/content.tsx

```diff
@@ -339,6 +346,52 @@ export default function Content({
                 />
               </div>
             </div>
+
+            <div id="tab-highlight-ai" className="tab-content hidden">
+              <div className="bg-slate-50 p-3 sm:p-4 rounded-md border border-slate-200">
+                <div className="p-4">
+                  <p className="text-sm text-slate-700 mb-4">
+                    Download and setup Highlight AI from{" "}
+                    <a
+                      href="https://highlightai.com"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="text-blue-600 hover:text-blue-800"
+                    >
+                      highlightai.com
+                    </a>
+                  </p>
+
+                  <p className="text-sm text-slate-700 mb-4">
+                    Enter this URL as SSE URL:{" "}
+                    <code className="bg-slate-200 px-2 py-1 rounded text-blue-700 break-words block my-3">
+                      {url}
+                    </code>
+                  </p>
+
+                  <div className="my-6">
+                    <img
+                      src="https://mintlify.s3.us-west-1.amazonaws.com/highlight-898689a5/images/sse-plugin-setup.png"
```

Comment:
Could you please replace this with a screenshot you took or an instructional image provided by HighlightAI?